### PR TITLE
Add COPR integration Makefile and add CI test for RPM building

### PR DESCRIPTION
**Pull request by jlebon**
```
commit 50831ebcaafb8162222f57b75411a1f92fd9a036
Date:   Tue Feb 15 13:41:55 2022 -0500

    Add COPR integration Makefile

    I'd like to enable auto-builds of this repo to
    https://copr.fedorainfracloud.org/coprs/g/CoreOS/continuous/ so it could
    eventually feed into
    https://github.com/coreos/fedora-coreos-tracker/issues/910.
```
```
commit 338c4db6bb80fd81a8e10e1b6ae39552f2d12c40
Date:   Tue Feb 15 13:45:30 2022 -0500

    workflows: add "RPM build" test

    Verify that we can build using purely Fedora packages. This will catch
    earlier any Rust packages we need to update or add in Fedora rather than
    find out at release time.

    It also ensures that the new continuous COPR builds will not break since
    we don't vendor there either.
```

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -98,3 +97,26 @@ jobs:
         run: cargo build
       - name: cargo test
         run: cargo test
+  test-rpm-build:
```
Comment (bgilbert) [inline .github/workflows/rust.yml]: I'd lean toward putting this in a separate workflow, since it's unrelated to the other jobs in this file.

**Review by bgilbert — APPROVED** [review]
We should require the job in branch protection settings after this merges.

**Comment by jlebon** [comment]
Updated!

File: .github/workflows/rpm.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+name: Rust
```
Comment (bgilbert) [inline .github/workflows/rpm.yml]: Copypaste error.

**Review by bgilbert — APPROVED** [review]

**Pull request merged**